kibana-reports/server/routes/lib/deliverReport.ts

```diff
@@ -61,27 +61,26 @@ export const deliverReport = async (
       ''
     )}/app/opendistro_kibana_reports#/report_details/${reportId}`;
 
-    const template = composeEmbeddedHtml(
+    const embeddedHtml = composeEmbeddedHtml(
       htmlDescription,
       originalQueryUrl,
       reportDetailUrl,
       reportName
     );
 
-    const deliveryBody = {
+    const reqBody = {
       ...deliveryParams,
-      htmlDescription: template,
+      htmlDescription: embeddedHtml,
       refTag: reportId,
     };
 
     // send email
-    const notificationResp = await callCluster(
-      notificationClient,
+    const notificationResp = await notificationClient.callAsInternalUser(
+      // @ts-ignore
```

Comment:
What's the warning that this `@ts-ignore` is silencing?

Reply:
overload error, but as a matter of fact, both callAsInternal user and callAsCurrentUser can take a string